### ISSUE TITLE
Bug List for current version of qbx_vehiclekeys

### ISSUE THREAD

**Issue by alberttheprince** [Qbox-project/qbx_vehiclekeys#94]
### Summary

List of issues 

### Reproduction

This is a list of current issues for Qbox Vehicle keys following the most recent update ([7c1299c](https://github.com/Qbox-project/qbx_vehiclekeys/commit/7c1299c6da850b73da61026f00f9173ebc4c7089)
)

**Issues:**
- ~Stepping out of the vehicle will stick the "search for keys" on your screen, even if you already have hotwired the car/searched for keys~
- Hot wiring/search for keys animation should just be the same
- hot wiring/searching for keys causes ped to leave car occasionally
- ~Search for keys stuck on screen (consistent issue) if you step out of vehicle~
- You can't turn off motorcycles once you steal them from peds
- You can't jack world spawn bicycles even if set as shared (see bikes at the beach for example)
- Carjacking ped will just return to their car once car jack is done, ped should run away in fear
- Ped can be carjacked over and over with no limit (just keep aiming over and over)
- Ped instantaneously pops out of the car when being jacked, stands there in an A pose, doesn't put hands up consistently
- ~Unable to cancel hotwiring in progress~

**Not bugs but maybe QoL:**
- Switch to progress bar (I think Qbox generally uses progress bar still, also the circle loading is just hard to read)
- pressing W will auto start the car (this should be config option, or disabled entirely requiring users to press G to start car)
- Cancel the animation of trying to start car when searching for keys hasn't started yet (ped will do car start anims, and other car anims before even search/hotwiring)
- Not sure what the difference between searching for keys/hotwiring are (clarify this in config)

### Expected behavior

N/A

### Actual behavior

N/A

### Additional context

N/A

### Current Version

7c1299c 

### Custom Resources

N/A

**Comment by alberttheprince** [comment]
> Stepping out of the vehicle will stick the "search for keys" on your screen, even if you already have hotwired the car/searched for keys

Fixed

> Search for keys stuck on screen (consistent issue) if you step out of vehicle

Fixed

 > You can't turn off motorcycles once you steal them from peds
 
This is now more broken than before. Pulling a ped out of a car requires you to search for keys, motorcycles still don't turn off if you steal them.

> Unable to cancel hotwiring in progress

Fixed now.

**Comment by alberttheprince** [comment]
All tech issues fixed, moving QoL to another issue
